velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -348,6 +348,58 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
       childNode);
 }
 
+core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
+    const ::substrait::SortRel& sortRel) {
+  core::PlanNodePtr childNode;
+  if (sortRel.has_input()) {
+    childNode = toVeloxPlan(sortRel.input());
+  } else {
+    VELOX_FAIL("Child Rel is expected in SortRel.");
+  }
+
+  const auto& inputType = childNode->outputType();
+
+  std::vector<core::FieldAccessTypedExprPtr> sortingKeys;
+  std::vector<core::SortOrder> sortingOrders;
+
+  const auto& sorts = sortRel.sorts();
+  sortingKeys.reserve(sorts.size());
+  sortingOrders.reserve(sorts.size());
+
+  for (const auto& sort : sorts) {
+    if (sort.sort_kind_case() == 2) {
```

Comment:
What is the for?

Reply:
Remove this check. Because this check has done in `sort.direction()` method.